app/components/RunPanel.vue

```diff
@@ -15,21 +21,33 @@ const props = defineProps<{
 
     <div class="flex-btwn">
       <small>HTTP {{ props.status }} ({{ props.durationInMs }} ms)</small>
-      <NuxtLink
-        :to="`/run/${props.runId}`"
-        class="run-permalink"
-        title="Share this run"
-        target="_blank"
-      >
-        🔗 Permalink
-      </NuxtLink>
+      <div class="panel-actions">
+        <button
+          class="toggle-raw-headers"
+          :title="showRawHeaders ? 'Hide raw headers' : 'Show raw headers'"
+          @click="toggleRawHeaders"
+        >
+          {{ showRawHeaders ? 'Hide Raw Headers' : 'Show Raw Headers' }}
+        </button>
```

Comment:
I think it's better to have a single toggle somewhere on the page, not one per panel. Please propose something with nice UI/UX. Don't hestitate to use a regular ol' styled radio button.